docs/modules/jenkins-shared-library/partials/odsComponentStageCopyImage.adoc

```diff
@@ -0,0 +1,66 @@
+// Document generated by render-adoc.go from odsComponentStageCopyImage.adoc.tmpl; DO NOT EDIT.
+
+Copies a source image into the project.
+
+This is useful to get images into the OpenShift registry so that release manager will accept all images.
+
+The primary intention is for helm charts so that external images can be imported.
+
+== Options
+
+[cols="1,2"]
+|===
+| Option | Description
+
+
+| *branch* +
```

Comment:
not needed

Reply:
Yes I think this is because src/org/ods/component/CopyImageOptions.groovy extends `Options`, which has the branch param. The docs are auto-generated from that. If you do not extend from `Options`, you should be able to get rid of it.